src/services/datahub-client.ts

```diff
@@ -163,4 +173,39 @@ export class HttpDatahubClient implements DatahubClient {
       };
     }
   }
+
+  public async getDailyVisitsByPeriod({
+    domainName,
+    dateFrom,
+  }: {
+    domainName: string;
+    dateFrom: Date;
+  }): Promise<DailyVisitsResult> {
+    try {
+      const response = await this.customerGet<{ periods: DailyVisits[] }>(
```

Comment:
No se si es buena idea establecer el tipo de la respuesta de DataHub, ya que no estamos manteniendo esos tipos actualizados con la estructura que Leo expone.

```suggestion
      const response = await this.customerGet<any>(
```

Reply:
En todos los metodos actuales tenemos eso, cambiamos los demas tambien?

Reply:
Puede ser, creo que en este caso usarlo tipado no nos ayuda mucho, ya que el tipo es arbitrario, no?

Ojo, no hablo acerca de la salida de los métodos de los servicios, en la que creo que los tipos nos ayudan mucho, sino de la salida de los requests HTTP que hacemos.

Reply:
Listo, cambiado